web/src/main/java/org/cbioportal/proxy/ProxyController.java

```diff
@@ -1,38 +1,74 @@
 package org.cbioportal.proxy;
 
-import org.springframework.http.HttpEntity;
-import org.springframework.http.HttpHeaders;
-import org.springframework.http.HttpMethod;
-import org.springframework.http.MediaType;
+import org.cbioportal.web.util.ResettableHttpServletRequestFilter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.*;
 import org.springframework.http.converter.StringHttpMessageConverter;
+import org.springframework.util.StringUtils;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 import org.springframework.web.client.RestTemplate;
 
+import javax.annotation.PostConstruct;
 import javax.servlet.http.HttpServletRequest;
 import java.net.URI;
 import java.net.URISyntaxException;
 import java.nio.charset.StandardCharsets;
 
 @RestController
 public class ProxyController {
+    @Value("${show.oncokb:true}")
+    private Boolean showOncokb;
+    @Value("${oncokb.token:}")
+    private String oncokbToken;
+    @Value("${oncokb.public_api.url:}")
+    private String oncokbPublicApiUrl;
+
+    private Logger LOG = LoggerFactory.getLogger(ProxyController.class);
 
     @RequestMapping("/**")
     public String proxy(@RequestBody(required = false) String body, HttpMethod method, HttpServletRequest request)
         throws URISyntaxException {
+        HttpHeaders httpHeaders = new HttpHeaders();
+        String contentType = request.getHeader("Content-Type");
+        if (contentType != null) {
+            httpHeaders.setContentType(MediaType.valueOf(contentType));
```

Comment:
It took some thinking for me to understand this --- the content type of the proxied request is being set to the same content type as the received request.